### PR TITLE
Update weixin app event type

### PR DESCRIPTION
**Pull request by jimexist**
- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by Jimexist** [comment]
@taoqf @AlexStacker PTAL

**Comment by AlexStacker** [comment]
Give a small reminder，please write the owner as follows:
https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package
If you don’t follow that, it doesn’t work.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4557,7 +4557,7 @@
 /types/webvr-api/                               @efokschaner
 /types/week/                                    @sindrenm
 /types/weighted/                                @ccitro
-/types/weixin-app/                              @taoqf @AlexStacker
+/types/weixin-app/                              @taoqf @AlexStacker @Jimexist
```
Comment (taoqf) [inline .github/CODEOWNERS]: It seems you don't need add your name here, add it into index.d.ts:line5. 
you may need read this https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package

**Review by taoqf — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (Jimexist) [reply .github/CODEOWNERS]: @taoqf thanks, updated

**Review by taoqf — APPROVED** [review]

**Pull request merged**